scio-smb/src/main/scala/org/apache/beam/sdk/extensions/smb/ParquetTypeFileOperations.scala

```diff
@@ -34,14 +34,13 @@ import java.nio.channels.{ReadableByteChannel, WritableByteChannel}
 
 object ParquetTypeFileOperations {
   val DefaultCompression = CompressionCodecName.GZIP
-  val DefaultConfiguration = new Configuration()
```

Comment:
this removal makes the PR fail MiMa. It seems that making these variables public is inconsistent with the behavior in the Java class, where it's package-private; I also don't think it has any usage since, as I mentioned, users don't typically interact with this class directly. So I think it's safe to merge--but I could also make MiMa happy by just setting `val DefaultConfiguration = null` and doing `Option(DefaultConfiguration).getOrElse(new Configuration())` in the calling method.

Reply:
@RustedBones any concern about merging this into 0.12.x when MiMa is failing? the impact of this bug should be zero so it could probably wait for 0.13.x release

Reply:
The `def` variant will only be called once anyway so wouldn't resolve the issue if I'm remembering the details correctly.

IMO the `null` variant makes the most sense in terms of being consistent with all the other IOs